### DISCUSSION TITLE
This is a great theme

### DISCUSSION THREAD

**Discussion by epogrebnyak** [gethinode/hinode#879] [General]
Hope discussion format allows to express gratitude to the author and contributor team for developing this theme. There is so few of clean and useful themes, and this one is exceptional. I maintain a site in Doks then
me which you mention as one of inspirations, but will start a new theme in Hinode, thanks again for providing it.

**Comment by markdumay** [comment]
Thanks for your kind words @epogrebnyak, appreciate it!

**Comment by epogrebnyak** [comment]
To elaborate a bit - there seems to be two scientific publishing systems - one is `myst/jupyterbook`, another is `quarto` (aims utlimately creating a PDF), actively maintained open source is `mkdocs-material` with a nice sposorware model, then a few docs themes per static generator (like starlight for astro), but many docs themes are forgotten and not maintained, also typography is ususally not that great. For Hugo we have Doks, and now Hinode. I think Hinode as a very strong usecase when I want something book-like with good typography and modern look.
